releases/net471/KnownIssues/510901-Apps-targeting-.NET-4.6.1-who-using-.NET-Standard-libraries-might-be-broken-on-.NET-4.7.1.md

```diff
@@ -0,0 +1,16 @@
+## Symptoms
+
+Applications that target the .NET Framework 4.6.1 and use .NET Standard libraries may experience <xref:TypeLoadExceptions> exceptions when running on .NET 4.7.1
```

Comment:
this needs to be \<xref:System.TypeLoadException>
.NET 4.7.1 --> the .NET Framework 4.7.1